Cubical/ZCohomology/Groups/Sn.agda

```diff
@@ -0,0 +1,422 @@
+{-# OPTIONS --cubical --no-import-sorts --safe #-}
+module Cubical.ZCohomology.Groups.Sn where
+
+open import Cubical.ZCohomology.Base
+open import Cubical.ZCohomology.Properties
+open import Cubical.ZCohomology.MayerVietorisUnreduced
+open import Cubical.ZCohomology.Groups.Unit
+open import Cubical.ZCohomology.KcompPrelims
+
+open import Cubical.Foundations.HLevels
+open import Cubical.Foundations.Function
+open import Cubical.Foundations.Prelude
+open import Cubical.Foundations.Pointed
+open import Cubical.Foundations.Isomorphism
+open import Cubical.Foundations.GroupoidLaws
+
+open import Cubical.HITs.Pushout
+open import Cubical.HITs.Sn
+open import Cubical.HITs.S1
+open import Cubical.HITs.Susp
+open import Cubical.HITs.SetTruncation renaming (rec to sRec ; elim to sElim ; elim2 to sElim2)
+open import Cubical.HITs.PropositionalTruncation renaming (rec to pRec ; elim to pElim ; elim2 to pElim2 ; ∥_∥ to ∥_∥₋₁ ; ∣_∣ to ∣_∣₋₁)
+open import Cubical.HITs.Nullification
+
+open import Cubical.Data.Sigma hiding (_×_)
+open import Cubical.Data.Int renaming (_+_ to _+ℤ_; +-comm to +ℤ-comm ; +-assoc to +ℤ-assoc)
+open import Cubical.Data.Nat
+open import Cubical.Data.Prod
+open import Cubical.HITs.Truncation renaming (elim to trElim ; map to trMap ; rec to trRec)
+open import Cubical.Data.Unit
+open import Cubical.Data.Group.Base renaming (Iso to grIso ; compIso to compGrIso ; invIso to invGrIso ; idIso to idGrIso)
+open import Cubical.Data.Group.Properties
+
+
+
+H⁰-Sⁿ≅ℤ : (n : ℕ) → grIso (coHomGr 0 (S₊ (suc n))) intGroup
```

Comment:
Ah yeah, this pretty much a one-liner... I didn't think of this. Nice! I will include it. I think I'll use the definition of connectedness in terms of mere existence of paths instead of the definition in the library in that case. Otherwise, if I use the definition of connectedness in the library, the proof will rely on PathIdTrunc from Truncation.Properties, which I suspect could slow things down a bit.